src/popup/modules/UserInterface.js

```diff
@@ -554,13 +555,12 @@ export function init() {
             // remove menu item if it has been added before
             browser.menus.remove(CONTEXT_MENU_SAVE_IMAGE);
 
-            return;
+            return undefined;
```

Comment:
This says not much and you could also have returned nothing AFAIK. However, what about specifically saying that the promise is resolved here:
```suggestion
            return Promise.resolve();
```